### ISSUE TITLE
Send only the Attachments to DevonThink

### ISSUE THREAD

**Issue by chellmann** [mailmate/devonthink-pro.mmbundle#2]
I would love to see a Command that just send the attached files (optionally filtered by filetype) to Devonthink.

I've get a lot of mails with a invoice attached - i don't want the complete mail as .eml file because devonthink can't search inside the attached PDF's

**Comment by matyaska** [comment]
I agree with chellmen. In alternative, the exposure of the save "attachments... functionality" in the rules would work (if with the ability of predetermine the destination directory). It would be fantastic is that command could export tags as well.

Thank you!

**Comment by freron** [comment]
Tagging the saved attachments? I guess that should be a general option for saving attachments (I'll make a note of that.)

@chellmann: Bundle commands cannot currently get the attachments of a message (at least not easily). MailMate has to support that before this can be supported.

**Comment by chellmann** [comment]
Hello,

long time no news here - is that feature (grabbing the attachements and send them to a third-party app.) on the todo list?

**Comment by the-pulli** [comment]
@chellmann I created a bundle to do that for DEVONthink:

https://github.com/the-pulli/DEVONthink-Attachments.mmBundle

Check it out and let me know what you think!
